[DOC] Add getting started guide for Sysadmins

surchs opened the pull request:
<!---
Until this PR is ready for review, you can include the [WIP] tag in its title, or leave it as a github draft.
-->






<!---
This is a suggested pull request template.
It's designed to capture information we've found to be useful in reviewing pull requests.

If there is other information that would be helpful to include, please don't hesitate to add it!
-->

<!-- Please indicate after the # which issue you're closing with this PR.
This is helpful for the maintainers AND will magically close the issue
when this pull request is merged!
If the PR closes multiple issues, includes "closes" before each one is listed.

You can also just link to other issues if necessary, e.g. "See also #1234".

https://help.github.com/articles/closing-issues-using-keywords
-->
Closes #13


<!-- Please give a brief overview of what has changed in the PR.
If you're not sure what to write, consider it a note to the maintainers
to indicate what they should be looking for when they review the pull request.
-->
Changes proposed in this pull request:

- new route "Getting Started", for now just for Sysadmin users

rmanaem commented on docs/start.md:
````diff
@@ -0,0 +1,207 @@
+with a name of `test_data`.
+
+```console
+curl -X POST -i -u "admin:admin" http://localhost:5820/admin/databases \
````
Since the password for `admin` is changed and we use the `NewPassword` in other instruction, I believe this should be `admin:NewPassword`

rmanaem commented on docs/start.md:
```diff
@@ -0,0 +1,207 @@
+
+## Launch the API and Graph stack
+
+Please [follow the instructions here](https://github.com/neurobagel/api/blob/main/README.md#docker) 
```
I'd say reference the [Local Installation](https://github.com/neurobagel/api/blob/main/README.md#local-installation) instead of just option 1 and then mention after following the initial instructions use the option 1. Could mention the [Troubleshooting](https://github.com/neurobagel/api/blob/main/README.md#troubleshooting) section as well.

rmanaem commented on docs/start.md:
````diff
@@ -0,0 +1,207 @@
+Upload the example files to the graph using this command:
+
+```console
+curl -u "admin:newPassword" -i -X POST http://localhost:5820/test_data \
````
`NewPassword`

rmanaem reviewed (changes requested):
Thanks for the PR @surchs! I made a few comments. I think to improve the docs we should document `deteils: Not found Not found' in the docs either here or in the API README.

surchs commented:
Thanks @rmanaem! Added the suggestions

> I think to improve the docs we should document `deteils: Not found Not found' in the docs either here or in the API README.

Added to the spec of this issue: https://github.com/neurobagel/api/issues/115
